test/e2e/util.go

```diff
@@ -489,24 +496,22 @@ func waitAWReadyQuiet(ctx *context, aw *arbv1.AppWrapper) error {
 	return waitAWPodsReadyEx(ctx, aw, int(aw.Spec.SchedSpec.MinAvailable), true)
 }
 
-
 func waitAWDeleted(ctx *context, aw *arbv1.AppWrapper, pods []*v1.Pod) error {
-	return waitAWPodsTerminatedEx(ctx, aw.Namespace, pods,0)
+	return waitAWPodsTerminatedEx(ctx, aw.Namespace, pods, 0)
 }
 
 func waitAWPending(ctx *context, aw *arbv1.AppWrapper) error {
 	return wait.Poll(100*time.Millisecond, ninetySeconds, awPodPhase(ctx, aw,
 		[]v1.PodPhase{v1.PodPending}, int(aw.Spec.SchedSpec.MinAvailable), false))
 }
 
-
 func waitAWPodsReadyEx(ctx *context, aw *arbv1.AppWrapper, taskNum int, quite bool) error {
-	return wait.Poll(100*time.Millisecond, ninetySeconds, awPodPhase(ctx, aw,
+	return wait.Poll(100*time.Millisecond, delaySeconds, awPodPhase(ctx, aw,
```

Comment:
Do we still need to do the 5 minute delay here or can we go back to 90 seconds after all the fixes you were able to do?

Reply:
Addressed!!

Reply:
lgtm